internal/oem/oem.go

```diff
@@ -244,6 +245,34 @@ alias gsutil="(docker images google/cloud-sdk || docker pull google/cloud-sdk) >
 		name:  "file",
 		fetch: file.FetchConfig,
 	})
+	configs.Register(Config{
+		name:  "sakuracloud",
+		fetch: sakuracloud.FetchConfig,
+		baseConfig: types.Config{
+			Systemd: types.Systemd{
+				Units: []types.SystemdUnit{
+					{
+						Enable: true,
+						Name:   "coreos-metadata-sshkeys@.service",
+					},
+					{
+						Enable: true,
+						Name:   "coreos-sakuracloud-network.service",
+						Contents: `[Unit]
```

Comment:
@crawford I moved the unit to https://github.com/coreos/coreos-overlay/pull/2460/files.